### DISCUSSION TITLE
User-Agent

### DISCUSSION THREAD

**Discussion by zahidunal** [FlareSolverr/FlareSolverr#1364] [General]
Isn't it time we used the updated user agent value? Can you help me how to achieve this? When this version is run user agent= "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36"

**Comment by ilike2burnthing** [comment]
Edit this line to more recent version:

https://github.com/FlareSolverr/FlareSolverr/blob/a2fe9e7776b4e2ead7ff20cf599e7efbaf11f89d/src/build_package.py#L28

and run from source (see readme), or update the Chromium folder by manually downloading a newer version.
